pandas/tests/extension/json/array.py

```diff
@@ -170,6 +170,9 @@ def _values_for_argsort(self):
         return np.array(frozen, dtype=object)[1:]
 
 
+JSONDtype.array_type = JSONArray
```

Comment:
Rather than pin this attribute ex-post, could it be a property defined directly in JSONDtype?